lib/crawler/coordinator.rb

```diff
@@ -355,20 +360,20 @@ def execute_task(crawl_task, follow_redirects: false)
       executor.run(crawl_task, follow_redirects:).tap do |crawl_result|
         events.url_fetch(url: crawl_task.url, crawl_result:, auth_type: crawl_task.auth_type)
         # get result of specific crawl if we are doing a url test command run
-        @url_test_results.push(crawl_result) if @url_test
+        @url_test_results.push(crawl_result) if @is_url_test
       end
     end
 
     # Process a crawl_result:
-    # - Extract canonical_url and add it to the backlog
-    # - Extract links contained in the page and add them to the backlog
+    # - Extract canonical_url and add it to the backlog (if applicable)
+    # - Extract links contained in the page and add them to the backlog (if applicable)
     # - Output the crawl_result to the sink
     def process_crawl_result(crawl_task, crawl_result)
       crawl_task_progress(crawl_task, 'processing result')
 
       # Extract and enqueue all links from the crawl result unless we are running a url test command
       start_time = Time.now
-      duration = Benchmark.measure { extract_and_enqueue_links(crawl_task, crawl_result) } unless @url_test
+      duration = Benchmark.measure { extract_and_enqueue_links(crawl_task, crawl_result) }
```

Comment:
Just for my understanding, why did we remove the `unless` here? Should we update the comment on L374 in that case too?

Reply:
Actually that guard clause could be a single line, I'll refactor that a bit

Reply:
I missed your comment about the L374 comment, also fixed